zarr/tests/test_core.py

```diff
@@ -2268,23 +2162,35 @@ def expected(self):
 
 class TestArrayWithFilters(TestArray):
 
-    @staticmethod
-    def create_array(read_only=False, **kwargs):
-        store = KVStore(dict())
-        dtype = kwargs.get('dtype')
+    compressor = Zlib(1)
+
+    def create_array(self, shape: Union[int, Tuple[int, ...]], **kwargs):
```

Comment:
Rather than re-implementing this whole method in the subclass, could you add `filters` to the base `TestArray` class attributes?

Reply:
done in [9d5b41f](https://github.com/zarr-developers/zarr-python/pull/1462/commits/9d5b41ffc6cfe0739becdaa373634779837d2938)